Sprint-1/exercises/random.js

```diff
@@ -4,6 +4,14 @@ const maximum = 100;
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 
 // In this exercise, you will need to work out what num represents?
+
+// FROM WHAT I CAN SEE. NUM IS A NUMBER GENERATED WITH THE RANDOM FUNCTION. NUM IS A VALUE BETWEEN 1 - 100.
 // Try breaking down the expression and using documentation to explain what it means
+
+//MATH.FLOOR MAKES SURE THAT A WHOLE NUMBER IS RETURNED. IT RETURNS THE SMALLEST WHOLE NUM POSSIBLE WHEN POSITIVE.
+//MATH.RANDOM RETURN A DECIMAL POINT LESS THAN 1
+// MAX-MIN +1 , We will get a number between 1 and 100 adding the min again makes sure that 100 is included in the range
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
```

Comment:
To test your understanding, how would you write an expression (without using any variable in the expression) that yields a random integer between -5 and 3 (including -5 and 3)?

Reply:
I would write an expression like this:

`return  Math.floor(Math.random() * (3 - (-5) + 1))  + (- 5);
`
or to simplify:
`return  Math.floor(Math.random() * (3 + 5 + 1))  - 5);
`
I'll just follow the basic principles that we have to use to randomly generate a number between a specific range.